frontend/src/Commons/Forms/FormModule/FormModule.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * @summary A Form Section, container for an Input Module
+ * @desc A Form Module is a component that takes a form section container,
+ *       containing a header and an Input Module
+ *
+ * @param InputModule: ComponentType<string> => Takes string name of a Component, InputModule
+ * @param name: string => name of the Form Section
+ * @param enable: boolean => expand or collapse form section upon render
+ * @param faIcon: IconDefinition => Font Awesome IconDefinition to be passed to FormHeader component
+ *
+ * @example:
+      <FormModule
+        InputModule={farmInformation}
+        name="Fields and Soil"
+        faIcon={faWheatAwn}
+        enable={false}
+      />
+ *
+ * @author @GDamaso
+ */
+import React, { useState } from 'react';
+import InputModuleInterface from 'src/Interface/InputModuleinterface.tsx';
```

Comment:
```suggestion
import InputModuleInterface from 'src/Interface/InputModuleInterface.tsx';
```
This file name should be corrected for its casing